latex/Research_Strategy.tex

```diff
@@ -190,6 +190,23 @@
 
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%SAMPL HISTORY
+\begin{wrapfigure}[20]{r}{7cm}
+%\vspace{-0.25in}
+\begin{centering}
+\includegraphics[width=\textwidth]{figures/history_v1.pdf}
+
+\end{centering}
+\footnotesize
+\caption{\label{figure:sampl5_logD}  
+\textbf{SAMPL historical participation~\cite{mobley_blind_2014-1}.} 
+Historical participation in SAMPL host-guest + solvation/distribution challenges has climbed rapidly, and we expect this trend to continue. The number of participating groups is shown in blue, and the number of submissions in green. The inset shows the diversity method types employed for the SAMPL4 hydration challenge, which is typical for SAMPL. 
+}
+\end{wrapfigure}
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
 Here, we \textbf{design a new series of SAMPL challenges specifically to maximize learning value to the community}.
```

Comment:
Can we change phrases like "intended to maximize learning value" to something more specific? I still find that phrase awkward. What are they learning?

What about "we design a new series of SAMPL challenges intended to maximize the ability of the modeling community to make fundamental improvements in modeling accuracy in essential subproblems for modeling protein-ligand interactions"?